rfc6962-bis.xml

```diff
@@ -1359,7 +1359,14 @@ but it is expected there will be a variety.
       </section>
       <section title="Auditing">
         <t>
-          Auditing is taking partial information about a log as input and verifying that this information is consistent with other partial information held. All clients described above may perform auditing as an additional function. The action taken by the client if audit fails is not specified, but note that in general if audit fails, the client is in possession of signed proof of the log's misbehavior.
+          Auditing ensures that the current published state of a log is reachable from previously published
+          states that are known to be good.  Audits are performed by monitors or TLS clients.
+        </t>
+        <t>
+          A benign, conformant log publishes a series of STHs over time, each derived from the previous STH and the submitted entries incorporated into the log since publication of the previous STH.  This can be proven through auditing of STHs.  SCTs returned to TLS clients can be audited by verifying against the accompanying certificate, and using Merkle Inclusion Proofs, against the log's Merkle tree.
+        </t>
+        <t>
+          The action taken by the auditor if an audit fails is not specified, but note that in general if audit fails, the auditor is in possession of signed proof of the log's misbehavior.
```

Comment:
And if you're going to spell this out, you need to also spell out SCTs as mentioned below.